php-practice/section5.php

```diff
@@ -217,11 +235,11 @@ public function getCount() {
 $userClickedMouseCounter->increment();
 $userClickedMouseCounter->increment();
 $userClickedMouseCounter->increment();
-echo $userClickedMouseCounter->getCount(); // What should this print out? Answer:
+echo $userClickedMouseCounter->getCount(); // What should this print out? Answer: 3 (since we passed the increment function 3 times)
```

Comment:
Good!

"Pass" isn't the word you're looking for here -- "call" or "invoke" are the most common words for executing a function. (Heh, "execute" is another one too.)

Check out [this W3Schools article](http://www.w3schools.com/js/js_function_invocation.asp) for JavaScript functions:

> You have already learned that the code inside a function will execute when "something" invokes it.
> The code in a function is not executed when the function is defined. It is executed when the function is invoked.
> Some people use the term "call a function" instead of "invoke a function".
> It is also quite common to say "call upon a function", "start a function", or "execute a function".

I'm not sure about that last line... usually I hear programmers say "call" or "invoke" -- sometimes "execute".